test/content-length.test.js

```diff
@@ -15,26 +14,25 @@ test('default 413 with bodyLimit option', t => {
     reply.send({ hello: 'world' })
   })
 
-  fastify.inject({
+  const response = await fastify.inject({
     method: 'POST',
     url: '/',
     body: {
       text: '12345678901234567890123456789012345678901234567890'
     }
-  }, (err, res) => {
-    t.error(err)
-    t.equal(res.statusCode, 413)
-    t.equal(res.headers['content-type'], 'application/json; charset=utf-8')
-    t.same(JSON.parse(res.payload), {
-      error: 'Payload Too Large',
-      code: 'FST_ERR_CTP_BODY_TOO_LARGE',
-      message: 'Request body is too large',
-      statusCode: 413
-    })
+  })
+  t.assert.ifError(response.err)
```

Comment:
`response.err` does not exist - we need to update the plan and remove this line

Reply:
I have updated the plan see commit here: [b5f4c7c](https://github.com/fastify/fastify/pull/5878/commits/b5f4c7c7b0ecbf5ae59c3f231ff86fa466415030), but it still fails